content/lessons/chapter-4/address-2.tsx

```diff
@@ -66,14 +71,16 @@ ${prevData?.data && 'compressedPublicKey = ' + "'" + dataObject + "'"}
     },
     constraints: [
       {
-        range: [9, 1, 9, 1],
+        range: [9, 1, 11, 1],
         allowMultiline: true,
       },
     ],
   }
 
   const python = {
-    program: `print("KILL")`,
+    program: `
```

Comment:
We need to update the solution in `address.tsx` in resources to reflect this change.